src/core/Notification/Notification.test.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import { Notification } from './Notification';
+import { axeTest } from '../../utils/test';
+
+describe('Error view tests', () => {
+  const ErrorNotification = (
+    <Notification
+      status="error"
+      className="notification-test-class"
+      closeText="Close"
+      smallScreen
+      labelText="Lorem ipsum dolor sit"
+    >
+      Testcontent
+    </Notification>
+  );
+
+  it('should have status and smallScreen prop specific classNames', () => {
+    const { container } = render(ErrorNotification);
+    expect(container.firstChild).toHaveClass('fi-notification--error');
+    expect(container.firstChild).toHaveClass('fi-notification--small-screen');
+  });
+  it('should have given className as props', () => {
+    const { container } = render(ErrorNotification);
+    expect(container.firstChild).toHaveClass('notification-test-class');
+  });
```

Comment:
We already have test for `className` prop, so this test is not needed.

Reply:
Fixed. Is now removed.